src/UIState/SeePh.cpp

```diff
@@ -0,0 +1,98 @@
+/**
+ * SeePIDConstants.cpp
+ */
+
+#include "SeePh.h"
+
+#include "Devices/DateTime_TC.h"
+#include "Devices/LiquidCrystal_TC.h"
+#include "Devices/PHControl.h"
+#include "Devices/PHProbe.h"
+#include "TC_util.h"
+
+void SeePh::loop() {
+  int32_t elapsedSeconds = ((millis() - startTime) / 3000) % 2;
```

Comment:
Perhaps a name like `oddEven` would be better since this value is not anything close to "elapsedSeconds".